starter-code/index.html

```diff
@@ -89,8 +89,8 @@
       &copy; 2016 Code Fellows | Static Blog | Happy Ipsum!
     </footer>
     <!-- Vendor Files -->
-    <!-- TODO: link to our new page.js library within vendors-->
-
+    <!-- TODONE: link to our new page.js library within vendors-->
+    <script src="https://cdn.rawgit.com/visionmedia/page.js/master/page.js"></script>
```

Comment:
This will work, but you have a page.js already in your vendor/scripts directory that you can use :)